test/inspector/test-async-hook-setup-at-inspect.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+common.skipIf32Bits();
+common.crashOnUnhandledRejection();
+const { NodeInstance } = require('../inspector/inspector-helper.js');
+const assert = require('assert');
+
+// Even with --inspect, the default async call stack depth is 0. We need a
+// chance to call Debugger.setAsyncCallStackDepth *before* activating the timer
+// for async stack traces to work.
```

Comment:
Here's the offending code if you're interested:

https://github.com/nodejs/node/blob/be63c26e8c5d9f337cc7349ea37dc5b312c9631d/deps/v8/src/inspector/v8-debugger.cc#L167-L176